src/main/java/uk/gov/hmcts/ccd/data/SecurityUtils.java

```diff
@@ -26,35 +31,42 @@ public HttpHeaders authorizationHeaders() {
         headers.add("user-roles", getUserRolesHeader());
 
         if (SecurityContextHolder.getContext().getAuthentication() != null) {
-            final ServiceAndUserDetails serviceAndUser = (ServiceAndUserDetails) SecurityContextHolder.getContext().getAuthentication().getPrincipal();
-            if (serviceAndUser.getPassword() != null) {
-                headers.add(HttpHeaders.AUTHORIZATION, serviceAndUser.getPassword());
-            }
+            headers.add(HttpHeaders.AUTHORIZATION, getUserBearerToken());
         }
         return headers;
     }
 
     public HttpHeaders userAuthorizationHeaders() {
-        final ServiceAndUserDetails serviceAndUser = (ServiceAndUserDetails) SecurityContextHolder.getContext().getAuthentication().getPrincipal();
         final HttpHeaders headers = new HttpHeaders();
-        headers.add(HttpHeaders.AUTHORIZATION, serviceAndUser.getPassword());
+        headers.add(HttpHeaders.AUTHORIZATION, getUserBearerToken());
         return headers;
     }
 
+    public UserInfo getUserInfo() {
+        return idamRepository.getUserInfo(getUserToken());
```

Comment:
Does this work? 
![image](https://user-images.githubusercontent.com/47384516/75338558-cf008080-5886-11ea-947f-69c977d31544.png)
You aren't getting the bearer token, just the token and the endpoint expect bearer. I checked the client and it doesnt seem to prepend "Bearer" to the header

Reply:
"Bearer" append in the  idamRepository  method https://github.com/hmcts/ccd-data-store-api/blob/9fdc0723f8c35c21b6d73e9da9d52fc5c28cdd7b/src/main/java/uk/gov/hmcts/ccd/security/idam/IdamRepository.java#L21

Reply:
ah I see